packages/sdk/src/dataProtector/sharing/subgraph/getRenters.ts

```diff
@@ -0,0 +1,56 @@
+import { gql, type GraphQLClient } from 'graphql-request';
+import { throwIfMissing } from '../../../utils/validators.js';
+import type {
+  GetRentersParams,
+  GraphQLRentersResponse,
+  Renters,
+} from '../../types.js';
+
+export async function getRenters({
+  graphQLClient,
+  protectedDataAddress = throwIfMissing(),
+}: {
+  graphQLClient: GraphQLClient;
+} & GetRentersParams): Promise<Renters[]> {
+  try {
+    const getRentersForProtectedDataQuery = gql`
+    query MyQuery {
+      protectedData(id: "${protectedDataAddress}") {
+        rentals {
```

Comment:
The common use-case will probably be to get **current** renters. Should we add a `endDate >= current date` filter?

Or a `includePastRentals: boolean` in parameters?

Reply:
yeah good questions. I prefer to include `includePastRentals: boolean` in parameters

Reply:
Commit: 102f00b72b2374e77dc32bccc48b41416fc41804